README.md

```diff
@@ -43,3 +36,152 @@ npm install -g webpack
 npm install
 ```
 
+## Server API
+
+### Add a property
+- POST api.localhost:8040/v1/homes
```

Comment:
all of us should be using `/listings` instead of '/homes' ?

Reply:
You're right, I'll get on changing that